methods.py

```diff
@@ -1465,14 +1460,14 @@ def format_key_value(v):
                 if godot_platform != "windows":
                     configurations += [
                         f'<ProjectConfiguration Include="editor|{proj_plat}">',
-                        f"  <Configuration>editor</Configuration>",
+                        "  <Configuration>editor</Configuration>",
```

Comment:
That's technically correct, but a bit sad to lose the alignment.

Reply:
Nothing we can do I'm afraid, seems to be a shortcoming of idiomatic Python.

Reply:
This could use a multiline f-string, but I'm not sure this would make things much better for 3 lines (the big drawback of multiline strings is losing the indentation, which also looks bad). So I guess it's fine as is.

Reply:
One other way of solving this would be to drop f-strings in this instance altogether:

```python
if godot_platform != "windows":
    configurations += [
        '<ProjectConfiguration Include="editor|' + proj_plat + '">',
        "  <Configuration>editor</Configuration>",
        "  <Platform>" + proj_plat + "</Platform>",
        "</ProjectConfiguration>",
    ]
```

Though that would probably raise a warning about not using f-strings, so there's no winning here.